### DISCUSSION TITLE
Network Frustration

### DISCUSSION THREAD

**Discussion by wabragg** [itzg/docker-minecraft-bedrock-server#179] [General]
Running 1.17.11.01 in Docker using a 'macvlan' to give it a dedicated IP.  I've got my port forward setup on the router and I can run the game offsite (let's say on McDonalds WiFi) and see the ping value.  I can also ping and play on the server on my local lan on my iPhone7.  But for the life of me, I can't get it to see the server or ping on the local lan on an iPad Pro.

Same subnet, same DNS server giving the address, same AP in the ceiling, same iOS version.

When running on the 'macvlan' driver, it doesn't show in the Friends section, but I setup a Server for it and that works fine on the iPhone ... just not the iPad.

I can see the broadcasts going out every 5 seconds on 192.168.100.255.19132 when I run 'tcpdump' on the host but oddly, some are for port 19132 and some are for port 65535 and some have an odd packet size with a bad checksum. (using -vv on tcpdump)

I'm thinking the network library they're using inside the program is junk or at the very least, doesn't play well with Docker.  All of my other containers on the 'macvlan' run just fine ... each with a dedicated IP.  I got tired of port collisions.

**Comment by jaydenthorup** [comment]
How is the local set up? I would think that you might need "nat reflection enabled" on your firewall rule, or whatever you have setup in the router. Just a thought, could be wrong and 2 years too late.
